### ISSUE TITLE
Missing `long_description` in python setup for PyPI

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#3497]
The PyPI publication for v7.3.0 failed due to missing package metadata see:

```
Checking dist/nucypher-7.3.0-py3-none-any.whl: FAILED
ERROR    `long_description` has syntax errors in markup and would not be        
         rendered on PyPI.                                                      
         No content rendered from RST source.                                   
WARNING  `long_description_content_type` missing. defaulting to `text/x-rst`.   
Checking dist/nucypher-7.3.0.tar.gz: PASSED with warnings
WARNING  `long_description_content_type` missing. defaulting to `text/x-rst`.   
WARNING  `long_description` missing.       
```

**Comment by derekpierre** [comment]
The following was removed from `setup.py` during `7.3.x` work.

```
    long_description_content_type="text/markdown",
    long_description_markdown_filename='README.md',
```

**Comment by manumonti** [comment]
so, can we close this? @KPrasch @derekpierre

**Comment by derekpierre** [comment]
> so, can we close this? @KPrasch @derekpierre

It's fixed for the future. However, this issue prevented v7.3.0 from being published to PyPI - which we skipped to get the release out, given our focus on docker images for nodes.

However, the latest publication of `nucypher` on pypi is still 7.2.0 - https://pypi.org/project/nucypher/.

We probably need to manually push a v7.3.0 release to pypi. We could:
1. checkout tags/v7.3.0
2. cherrypick https://github.com/nucypher/nucypher/pull/3509/commits/eda898692488a6e6a7f9f0edbd43cacaa60c2ec2, 3. perform a manual release to pypi.

@KPrasch does that make sense? Are you able to take care of that?

**Comment by derekpierre** [comment]
7.3.0 is now published after cherry-picking the commit from #3509 a local copy of `tags/v7.3.0` - https://pypi.org/project/nucypher/.
